<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>651</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Zašífrovat</string>
    </property>
    <widget class="QLineEdit" name="inputZasifrovat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="zasifrovat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Zašifrovat</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Zadej text</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="outputZasifrovat">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>381</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>→</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>651</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Dešifrovat</string>
    </property>
    <widget class="QLineEdit" name="inputDesifrovat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="desifrovat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Dešifrovat</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Zadej text</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="outputDesifrovat">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>381</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>→</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="puvodniText">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Původní text</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rozdelit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>114</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Rozdělit po 5 znacích</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="klicA">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Klíč A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Klíč B</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="klicB">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="error">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
